src/Validator.ts

```diff
@@ -54,29 +64,29 @@ export default (data: ArrayBuffer | ArrayBufferLike): ValidationResults => {
   const chCount = header.getUint32(10, true);
 
   validationResult.frameCount = header.getUint32(14, true);
+  validationResult.channelCount = chCount
   validationResult.stepTime = header.getUint8(18);
 
   const compressionType = header.getUint8(20);
 
   if(magic !== 'PSEQ' || start < 24 || validationResult.frameCount < 1 || validationResult.stepTime < 15 || minor !== 0 || major !== 2) {
-    validationResult.error = 'Unknown file format, expected FSEQ v2.0';
-    return validationResult;
+    validationResult.errors!.push(ErrorType.FileFormat);
+    return validationResult
   }
```

Comment:
Since there are error types, I think it might make sense to break the `or` statements into separate errors.

Reply:
I think that would make sense, too. Probably would be better to do that outside of this PR, to keep things simple.